usaspending_api/api_contracts/contracts/v2/search/spending_by_award_count.md

```diff
@@ -120,6 +120,10 @@ This endpoint takes award filters, and returns the number of awards in each awar
 + `exclude`: `3333` (optional, array[string])
 
 ### TASCodeObject (object)
++ `require`: [[`091`]] (optional, array[array[string]])
++ `exclude`: [[`091`, `091-0800`]] (optional, array[array[string]])
+
+### TreasuryAccountComponentsObject (object)
```

Comment:
Does the `AdvancedFilterObject` object need to include these changes?

Reply:
Yes. @BrianZito please add `TreasuryAccountComponentsObject` to the object